LICENCE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 Nelly Telli
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
```

Comment:
- [ ] I noticed that you have multiple `LICENSE` files in your repository, which can cause confusion for contributors. To ensure that your project remains organized and easy to navigate, it's best to remove the duplicate `LICENSE` file.

Reply:
@lily4178993 please you have not attended to this requirement by your previous reviewer, make sure to remove duplicate files as it is unprofessional to have it the way it currently is in your branch :+1: